src/charm.py

```diff
@@ -109,11 +150,34 @@ def _install_cli_resources(self):
             log.exception("CLI binaries could not be installed.")
             return
 
+    def _install_service(self, service_file_path):
+        try:
+            service_path = Path("/etc/systemd/system")
+            shutil.copy(service_file_path, service_path)
+            subprocess.run(["systemctl", "daemon-reload"])
+        except subprocess.CalledProcessError:
+            log.exception("Failed to reload systemd daemons.")
+        except OSError:
+            log.exception("Destination folder: {service_path} is not writable.")
+
+    def _manage_port_forward_service(self, enable=False):
+        try:
+            if enable:
+                subprocess.run(["systemctl", "enable", PORT_FORWARD_SERVICE], check=True)
+                subprocess.run(["systemctl", "start", PORT_FORWARD_SERVICE], check=True)
+            else:
+                subprocess.run(["systemctl", "disable", PORT_FORWARD_SERVICE], check=True)
+                subprocess.run(["systemctl", "stop", PORT_FORWARD_SERVICE], check=True)
```

Comment:
ahh!  here's you're using `check=True`

you can replace with `check_call` if you'd like since you're not using the output